<ui version="4.0" >
 <class>QRL_LogsManager</class>
 <widget class="QDialog" name="QRL_LogsManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>450</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>450</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Logs Manager</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="qrtailab.qrc" >
    <normaloff>:/icons/log_icon.xpm</normaloff>:/icons/log_icon.xpm</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="helpButton" >
       <property name="text" >
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>Close Manager</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QListWidget" name="logListWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="baseSize" >
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="Log" >
      <attribute name="title" >
       <string>Log</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QToolBox" name="toolBox" >
         <property name="currentIndex" >
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_2" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>156</width>
            <height>225</height>
           </rect>
          </property>
          <attribute name="label" >
           <string>Saving</string>
          </attribute>
          <layout class="QGridLayout" >
           <item row="3" column="1" >
            <widget class="QLabel" name="label_2" >
             <property name="text" >
              <string>Filename</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1" colspan="2" >
            <widget class="QProgressBar" name="saveProgressBar" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="value" >
              <number>0</number>
             </property>
             <property name="textVisible" >
              <bool>true</bool>
             </property>
             <property name="format" >
              <string>%p%</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1" >
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>Time [s]:</string>
             </property>
            </widget>
           </item>
           <item row="11" column="1" >
            <widget class="QPushButton" name="savePushButton" >
             <property name="toolTip" >
              <string>start saving</string>
             </property>
             <property name="text" >
              <string>Save</string>
             </property>
             <property name="checkable" >
              <bool>true</bool>
             </property>
             <property name="autoDefault" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="11" column="2" >
            <widget class="QPushButton" name="stopPushButton" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="toolTip" >
              <string>stop saving</string>
             </property>
             <property name="text" >
              <string>stop</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QLabel" name="label_3" >
             <property name="text" >
              <string>Directory</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2" >
            <widget class="QLineEdit" name="dirLineEdit" >
             <property name="toolTip" >
              <string>Directory in which the file will be saved</string>
             </property>
             <property name="text" >
              <string>./</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2" >
            <widget class="QLineEdit" name="fileLineEdit" >
             <property name="toolTip" >
              <string>Filename</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2" >
            <widget class="QwtCounter" name="timeCounter" >
             <property name="toolTip" >
              <string>Time duration of the next saving</string>
             </property>
             <property name="numButtons" >
              <number>0</number>
             </property>
             <property name="minValue" >
              <double>0.000000000000000</double>
             </property>
             <property name="maxValue" >
              <double>1000000.000000000000000</double>
             </property>
             <property name="value" >
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="2" >
            <widget class="QPushButton" name="dirPushButton" >
             <property name="toolTip" >
              <string>set directory for saving</string>
             </property>
             <property name="text" >
              <string>Set Dir</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtCounter</class>
   <extends>QWidget</extends>
   <header>qwt_counter.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qrtailab.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>QRL_LogsManager</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>304</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel" >
     <x>394</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
